doc/source/container_builder.rst

```diff
@@ -0,0 +1,282 @@
+Building in a Self-Contained Environment
+========================================
+
+.. topic:: Abstract
+
+   Users building images with KIWI face problems if they want
+   to build as non root user or can't install KIWI on their
+   system due to inappropriate build host requirements or
+   project policies.
```

Comment:
I'm not sure if I understood the above sentence correctly. So there are two problems, right? This little "or" is a bit confusing. Maybe better start with this:

```Users building images with KIWI face two problems: 

```